articles/iot-edge/how-to-configure-api-proxy-module.md

```diff
@@ -16,12 +16,14 @@ monikerRange: ">=iotedge-2020-11"
 
 [!INCLUDE [iot-edge-version-202011](../../includes/iot-edge-version-202011.md)]
 
-There are several scenarios where you might want to deploy multiple services on an IoT Edge device, all of which support HTTPS protocol and bind to port 443. This is especially relevant in hierarchical deployments of IoT Edge devices in ISA-95 based network isolated architectures such as those described in [Network isolate downstream devices](how-to-connect-downstream-iot-edge-device.md#network-isolate-downstream-devices), as the clients on the child devices are not able to connect to the cloud directly. For example, to allow child IoT Edge devices to pull Docker images requires deploying a Docker registry module, and to allow uploading blobs requires deploying a Blob Storage module on the same IoT Edge device. Both these services use HTTPS for communication. The API Proxy enables such deployments on an IoT Edge device. Instead of each service, the API Proxy module binds to port 443 on the host device and routes the request to the right service module running on that device as per user configurable rules. The individual services are still responsible for handling the requests, including authenticating and authorization of the clients. Without the API Proxy, each service module would have to bind to a separate port on the host device, requiring tedious and error prone configuration change on each child device connecting to the parent IoT Edge device. 
+This article walks through the configuration options for the API proxy, so you can customize the module to support your gateway hierarchy requirements.
 
-This article walks through the configuration options for the API Proxy, so that you can customize the module to support your gateway hierarchy requirements.
+You might want to deploy multiple services on an IoT Edge device in several scenarios, all of which support the HTTPS protocol and bind to port 443. This is especially relevant in hierarchical deployments of IoT Edge devices in ISA-95-based network isolated architectures like those described in [Network isolate downstream devices](how-to-connect-downstream-iot-edge-device.md#network-isolate-downstream-devices) because the clients on the child devices can't connect directly to the cloud.
```

Comment:
@ktoliver I think rearranging this first sentence changed the meaning. It now sounds like "several scenarios support the HTTPS protocol and bind to port 443" where the important information here is that "multiple services support the HTTPs protocol and bind to port 443" 

How about, "The API proxy module simplifies communication for IoT Edge devices when multiple services are deployed that all support HTTPS protocol and bind to port 443."

Reply:
Yes, I'll make that change. I was trying to get around the "there are" sentence construction, which the Microsoft Writing Style Guide discourages because it's passive. Thanks!